src/bicep/mlz.bicep

```diff
@@ -487,6 +487,8 @@ param logAnalyticsWorkspaceLocation string = operationsLocation
 param logAnalyticsWorkspaceCappingDailyQuotaGb int = -1
 param logAnalyticsWorkspaceRetentionInDays int = 30
 param logAnalyticsWorkspaceSkuName string = 'PerGB2018'
+@description('When set to "True", enables Microsoft Sentinel within the MLZ Log Analytics workspace.')
+param enableSentinel bool = false
```

Comment:
let's name this `deploySentinel` for consistency -- the rest of the on/off switches are `deploy{X}`

Reply:
Yes sir, just updated the branch as requested.  Thank you!